packages/client/modules/meeting/components/ReflectTemplateListPublic.tsx

```diff
@@ -49,7 +49,7 @@ export default createFragmentContainer(ReflectTemplateListPublic, {
         id
         meetingSettings(meetingType: retrospective) {
           ... on RetrospectiveMeetingSettings {
-            publicTemplates(first: 20)
+            publicTemplates(first: 50)
```

Comment:
We have quite a few template options now. Gareth mentioned that he'll create an issue where we add a search box so users can find the template they're looking for more easily.

Reply:
whoa! that's crazy to think about